controllers/flp/flp_test.go

```diff
@@ -712,7 +712,6 @@ func TestConfigMapShouldDeserializeAsJSONWithLokiStack(t *testing.T) {
 		"DstK8S_Type",
 		"K8S_FlowLayer",
 		"_RecordType",
-		"FlowDirection",
```

Comment:
This need to be restored since we keep FlowDirection anyways now

Reply:
restored